templates/base.html

```diff
@@ -15,11 +15,11 @@
       gtag('config', 'UA-87887700-1', {'anonymize_ip': true});
     </script>
     {% endif %}
-    <title>{% block page-title %}{% block page-subtitle %}{% if page %}{{ page.meta_title|add:" | "}}{% endif %}{% endblock %}The Center for Digital Humanities at Princeton{% endblock %}</title>
+    <title>{% block page-title %}{% if page %}{{ page.seo_title }}{% endif %}{% endblock %} – CDH@Princeton</title>
```

Comment:
Ah, this is nice to handle in one place. Is that why all the others are being removed? If so, that's great.

Should this not be `page.seo_title` if set, otherwise `page.title` ? Suspect that most of our pages do _not_ have an seo specific title set.

Also, should the CDH@Princeton be overridable? seems like you wouldn't want it included on e.g. the home page. (Although that might be the _only_ place where it matters... 🤔 )

Reply:
And... I know you want to keep this short, but I'm not convinced CDH@Princeton is meaningful/recognizable enough. Is it really so terrible to keep the site name spelled out? Can't people with screen readers skip over it 
 once they hear it starts repeating?

Reply:
The spelled-out version is 47 characters, which is ~60% of our allowed title space. That seems like too much real estate to give up on every page, especially if Google may just add it to search results on its own anyway.

Reply:
ok, let's try it. we can always tweak if we need to